src/llmq/quorums_blockprocessor.cpp

```diff
@@ -188,14 +188,9 @@ bool CQuorumBlockProcessor::ProcessCommitment(int nHeight, const uint256& blockH
         return state.DoS(100, false, REJECT_INVALID, "bad-qc-block");
     }
 
-    // index of quorumHash already checked
+    // index of quorumHash (and commitment signature) already checked
```

Comment:
About 7cff2168:
Future idea, we could create a cache of special tx payloads, so we don't have to parse them twice (inside `CheckSpecialTx` and in `GetCommitmentsFromBlock` for the complete block validation) and we store a "IsVerified" bool inside it as well.

Reply:
Sure, there is room for optimization there in the future.